index.html

```diff
@@ -1,14 +1,223 @@
 <!DOCTYPE HTML>

```

Comment:
This is an overarching comment for this section. Looking at the Remock project description and screen shots, I originally see that this page is missing at least 6 buttons. Two for each of the larger post bodies, and three more on the smaller post sidebar sections. Upon looking further I discovered that you did indeed have buttons, but they are completely invisible. This is a big UX issue that will need to be resolved. 

A good start would be to giving these a `background-color: gray;` or something similar so you can at least see that the buttons exist when visiting the page.